modules/manage/pages/manage-security/security-management-overview.adoc

```diff
@@ -38,3 +38,83 @@ Information is provided in xref:manage:manage-logging/manage-logging.adoc[Manage
 This is described in xref:manage:manage-security/manage-sessions.adoc[Manage Sessions].
 
 See the navigation panel at the left, for details of additional management procedures documented in this section.
+
+[#security-checklist]
+== Security Checklist
+
+The security checklist below should be reviewed and used in the set-up and maintenance of a Couchbase-Server cluster.
+
+Optionally, the link:{attachmentsdir}/manage-security/cb7SecurityChecklist.pdf[checklist^] can be accessed as a PDF file.
+
+
+[cols="5,15"]
+|===
+|
+Access control
+a|
+* [ ] Create unique user accounts for each individual and application that accesses the platform.
+
+* [ ] Implement Role-Based Access Control and assign roles following a principle of least privilege
```

Comment:
```suggestion
* [ ] Implement Role-Based Access Control and assign roles following a principle of least privilege.
```

Reply:
I agree. But I've been asked to replicate the checklist exactly, so I've limited myself just to that.